src/app/components/elements/modals/RequiredAccountInformationModal.tsx

```diff
@@ -61,9 +60,7 @@ const RequiredAccountInfoBody = () => {
         }
     }
 
-    const allUserFieldsAreValid =
-        (isPhy && validateUserContexts(initialUserContexts)) ||
-        (isCS && validateUserSchool(initialUserValue) && validateUserGender(initialUserValue) && validateUserContexts(initialUserContexts));
```

Comment:
Just checking: there are a few uses of `isCs` in this and other files. Is the plan to remove `isCs` in a different ticket? If not, it would be worth adding a new ticket for it.

Reply:
I'm removing isCS and isPhy as part of the siteSpecific removal as that seemed to make the most sense.  There's a little bleed over between tickets (some siteSpecific was removed in this ticket) but siteSpecific, isCS and isPhy will be totally gone in the next ticket.